src/DependencyInjection/PsyshExtension.php

```diff
@@ -12,25 +12,34 @@
 namespace Fidry\PsyshBundle\DependencyInjection;
 
 use Psy\Command\Command;
+use Symfony\Bundle\FrameworkBundle\Test\TestContainer;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader;
 use Symfony\Component\DependencyInjection\Reference;
 use Symfony\Component\ExpressionLanguage\Expression;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
-use Symfony\Bundle\FrameworkBundle\Test\TestContainer;
+use function array_merge;
+use function is_string;
+use function sprintf;
+use function strpos;
+use function substr;
 
 /**
  * This is the class that loads and manages your bundle configuration.
  *
  * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
  *
  * @author Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * @private
  */
 final class PsyshExtension extends Extension implements PrependExtensionInterface
 {
-    public function prepend(ContainerBuilder $container)
+    private const CONFIG_DIR = __DIR__.'/../../resources/config';
+
+    public function prepend(ContainerBuilder $container): void
     {
         if (class_exists(TestContainer::class)) {
```

Comment:
Actually we can just copy the two services we need from `test.xml` instead of loading the whole file or forcing the test config. It appears to be working fine and less intrusive